<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Gestionnaire de Jeu de Rôle</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="statButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(70, 70, 70);
color: rgb(255, 255, 255);
selection-background-color: rgb(154, 154, 154);</string>
          </property>
          <property name="text">
           <string>Ajouter une stat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="itemWindowButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(70, 70, 70);
color: rgb(255, 255, 255);
selection-background-color: rgb(154, 154, 154);</string>
          </property>
          <property name="text">
           <string>Gestion des items</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="skillWindowButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(70, 70, 70);
color: rgb(255, 255, 255);
selection-background-color: rgb(154, 154, 154);</string>
          </property>
          <property name="text">
           <string>Gestion des skills</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="entityWindowButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(70, 70, 70);
color: rgb(255, 255, 255);
selection-background-color: rgb(154, 154, 154);</string>
          </property>
          <property name="text">
           <string>Gestion des entitées</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="backgroundButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(70, 70, 70);
color: rgb(255, 255, 255);
selection-background-color: rgb(154, 154, 154);</string>
          </property>
          <property name="text">
           <string>Choisir un terrain</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>657</width>
           <height>557</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="mapDisplay">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouvelle_partie"/>
    <addaction name="actionCharger_une_partie"/>
    <addaction name="actionEnregistrer_la_partie"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionComment_utiliser_l_application"/>
    <addaction name="actionA_propos"/>
    <addaction name="actionContact"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAide"/>
  </widget>
  <action name="actionNouvelle_partie">
   <property name="text">
    <string>Nouvelle partie</string>
   </property>
  </action>
  <action name="actionCharger_une_partie">
   <property name="text">
    <string>Charger une partie</string>
   </property>
  </action>
  <action name="actionEnregistrer_la_partie">
   <property name="text">
    <string>Enregistrer la partie</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionComment_utiliser_l_application">
   <property name="text">
    <string>Comment utiliser l'application</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionContact">
   <property name="text">
    <string>Contact</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
